aea/cli/fetch.py

```diff
@@ -33,48 +33,61 @@
     try_to_load_agent_config,
 )
 from aea.cli.registry.fetch import fetch_agent
-from aea.configurations.base import PublicId
+from aea.configurations.base import DEFAULT_AEA_CONFIG_FILE, PublicId
 from aea.configurations.constants import DEFAULT_REGISTRY_PATH
 
 
 @click.command(name="fetch")
 @click.option("--local", is_flag=True, help="For fetching agent from local folder.")
+@click.option(
+    "--alias", type=str, required=False, help="Provide a local alias for the agent.",
```

Comment:
Now it is possible to fetch the same agent multiple times by giving it a local alias!